### PR TITLE
Add ALUMNI.md file

### PR DESCRIPTION
**Pull request by znewman01**
Signed-off-by: Zachary Newman <zjn@chainguard.dev>

### CODE REVIEW

**Comment by dlorenc** [comment]
LGTM!

I made my own :) https://github.com/dlorenc/alumni/blob/main/ALUMNI.md

File: ALUMNI.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Alumni
+
+Thanks to the following folks who used to maintain Cosign!
+
+- @dlorenc
```
Comment (cpanato) [inline ALUMNI.md:5]: I would drop the @ and sort alphabetical and mention that is sorted in the inital text

**Review by hectorj2f — DISMISSED** [review]
Lgtm

File: ALUMNI.md (hunk 1 above)
Comment (znewman01) [reply ALUMNI.md]: I like the `@` to make it clear these are handles; I added links to GitHub profiles as well.

But sorting is a good idea, I've done that.

**Review by cpanato — APPROVED** [review]

**Pull request merged**